frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineRunItem.tsx

```diff
@@ -27,11 +28,13 @@ const PipelineRunItem: React.FC<PipelineRunItemProps> = ({ pipelineRun }) => {
   const logDetails = getLogSnippet(pipelineRun);
 
   return (
-    <li className="list-group-item">
+    <li className="odc-pipeline-run-item list-group-item">
       <Grid hasGutter>
         <GridItem span={6}>
           <Link to={`${path}`}>{name}</Link>
-          {lastUpdated && <span className="text-muted">&nbsp;({fromNow(lastUpdated)})</span>}
+          {lastUpdated && (
+            <span className="odc-pipeline-run-item__time text-muted">({fromNow(lastUpdated)})</span>
+          )}
```

Comment:
By removing `&nbsp;` you've removed the separation between text:
![image](https://user-images.githubusercontent.com/14068621/92785631-92edc280-f375-11ea-85b7-8f9233937a6c.png)

You need to use `{' '}`

Reply:
Doesn't work with the space, it gets trimmed off. I tried to achieve the behavior by using a right-padding on previous element.
Let me know if that's fine.